csxtools/utils.py

```diff
@@ -178,7 +178,10 @@ def get_images_to_3D(images, dtype=None):
 
 def _get_images(header, tag, roi=None):
     t = ttime.time()
-    images = header.db.get_images(header, tag)
+    if isinstance(header, (list, tuple)):
```

Comment:
Can you change this to use `header.data` instead?

Reply:
thanks for suggestion on new api. updated